syntax = "proto3";

import "akkaserverless/annotations.proto";

package com.example;

message ThingState {
    string thing_id = 1 [(akkaserverless.field).entity_key = true];
    string name = 2;
}
